spec.html

```diff
@@ -6211,6 +6222,8 @@
         1. Add _pending_ at the back of the Job Queue named by _queueName_.
         1. Return NormalCompletion(~empty~).
       </emu-alg>
+
+      <emu-note>For host environments consisting of multiple continents, when those host environments enqueue their own jobs, they should generally specify which continent's job queues they are enqueuing on.</emu-note>
```

Comment:
OK, I got this a bit wrong, I see. PendingJobs have a [[Realm]], and you could argue that the job queue chosen is implicitly the one belonging to the [[Realm]]'s parent continent. The [[Realm]] is chosen based on the running execution context. I could revise this note to something like

> For host environments consisting of multiple continents, each with their own set of job queues, it can be helpful to realize that EnqueueJob always enqueues in a job queue belonging to the running execution context's realm's continent.

or even just

> EnqueueJob always enqueues in a job queue belonging to the running execution context's realm's continent.

The main thing I am clarifying here (indeed, the main thing I am clarifying with this whole PR) is that in a host environment with multiple continents, I want it to be obvious where EnqueueJob("ScriptJobs", ...) is actually happening.

Now I need to figure out how to set the running execution context from within HTML... should be doable.